deployment/roles/metron_streaming/defaults/main.yml

```diff
@@ -34,3 +39,6 @@ storm_topologies:
     - "{{ metron_directory }}/config/topologies/yaf/remote.yaml"
     - "{{ metron_directory }}/config/topologies/pcap/remote.yaml"
     - "{{ metron_directory }}/config/topologies/enrichment/remote.yaml"
+hdfs_retention_days: 30
+hdfs_bro_purge_cronjob: "yarn jar {{ metron_directory }}/lib/{{ metron_dataloads_jar }} org.apache.metron.dataloads.bulk.HDFSDataPruner -f {{ hdfs_url }} -g '/apps/metron/enrichment/indexed/bro_doc/*enrichment-*' -s $(date -d '{{ hdfs_retention_days }} days ago' +%m/%d/%Y) -n 1 >> /var/log/bro-purge/cron-bro-purge.log 2>&1"
```

Comment:
Because `/usr/metron/0.1BETA/bin/purge_hdfs.sh -f ... -g ... -s ...` is a lot easier to run outside of automated scripts than remembering `yarn jar /usr/metron/0.1BETA/lib/Metron-DataLoads-0.1BETA.jar org.apache.metron.dataloads.bulk.HDFSDataPruner -f ... -g ... -s ...`